packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts

```diff
@@ -0,0 +1,130 @@
+import { FontSizes, FontWeights, HighContrastSelector, HighContrastSelectorBlack } from '@fluentui/react';
+import { IGaugeChartStyleProps, IGaugeChartStyles } from './GaugeChart.types';
+
+export const getStyles = (props: IGaugeChartStyleProps): IGaugeChartStyles => {
+  const { theme, chartValueSize, width, height, className, lineColor, toDrawShape } = props;
+
+  return {
+    root: [theme.fonts.medium, 'ms-GaugeChart', { width }, className],
+
+    chart: {
+      display: 'block',
+      width,
+      height,
+    },
+
+    limits: {
+      fontSize: FontSizes.small,
+      fontWeight: FontWeights.semibold,
+      fill: theme.palette.neutralPrimary,
+    },
+
+    chartValue: {
+      fontSize: chartValueSize,
```

Comment:
What is the default value for chartValueSize

Reply:
24